### DISCUSSION TITLE
Custom fonts in web app version?

### DISCUSSION THREAD

**Discussion by rauberdaniel** [typst/typst#1234] [Ideas]
With the CLI now supporting custom font directories, I would really appreciate the ability to upload custom ttf/otf fonts to the web app version in order to create documents according to a specific corporate identity.

**Comment by xkevio** [comment]
You can already use custom fonts in the web app. Just upload your ttf file and set the font with its name but without the extension.

**Reply by rauberdaniel** [reply]
You’re right, that works! Thats awesome, I don’t think thats documented anywhere though yet, right?
